### PR TITLE
add jenkins build

### PR DESCRIPTION
**Pull request by BrandonArp**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        script {
+          target = "verify"
+        }
+      }
+    }
+    stage('Setup release') {
+      when { buildingTag(); not { changeRequest() }  }
+      steps {
+        script {
+          target = "deploy -P release -P rpm --settings settings.xml"
```
Comment (vjkoskela) [inline Jenkinsfile:35]: No rpm

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        sh 'gpg --batch --import ism.key'
+      }
+    }
+    stage('Build') {
+      steps {
+        withCredentials([usernamePassword(credentialsId: 'jenkins-dockerhub', usernameVariable: 'DOCKER_USERNAME', passwordVariable: 'DOCKER_PASSWORD'),
+            usernamePassword(credentialsId: 'jenkins-ossrh', usernameVariable: 'OSSRH_USER', passwordVariable: 'OSSRH_PASS'),
+            string(credentialsId: 'jenkins-gpg', variable: 'GPG_PASS')]) {
+          withMaven {
+            sh "./jdk-wrapper.sh ./mvnw $target -P rpm -U -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Ddocker.verbose=true"
```
Comment (vjkoskela) [inline Jenkinsfile:46]: Same

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          }
+        }
+      }
+    }
+    stage('GitHub release') {
+      when { buildingTag(); not { changeRequest() }  }
+      steps {
+        withCredentials([usernamePassword(credentialsId: 'brandonarp-github-token', usernameVariable: 'GITHUB_USERNAME', passwordVariable: 'GITHUB_TOKEN')]) {
+          sh "github-release release --user ${org} --repo ${repo} --tag ${TAG_NAME}"
```
Comment (vjkoskela) [inline Jenkinsfile:55]: No GitHub release on pom

File: kubes-pod.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      value: "tcp://localhost:2376"
+    - name: "DOCKER_CERT_PATH"
+      value: "/certs/client"
+    - name: "_JAVA_OPTIONS"
+      value: "-Duser.home=/home/jenkins"
+    - name: "JDKW_VERBOSE"
+      value: "1"
+    - name: "HOME"
+      value: "/home/jenkins"
+    image: "docker.arpnetworking.com/build-base"
```
Comment (vjkoskela) [inline kubes-pod.yaml:32]: Is this right for ISM?

Comment (BrandonArp) [reply kubes-pod.yaml:32]: This is right. This is the docker container in the build system that is used as the base

**Pull request merged**